modules/@apostrophecms/page-type/index.js

```diff
@@ -292,6 +293,13 @@ module.exports = {
   },
   methods(self) {
     return {
+      addManagerModal() {
```

Comment:
This makes the modal a "sleeper" in the modals Vue app for purposes of the admin bar. So I think you can remove this if your feature works without it.